routes/api.php

```diff
@@ -14,9 +15,15 @@
 |
 */
 
```

Comment:
there's no need to redefine the login routes, it's already defined in `routes/auth.php`

Reply:
Can you explain this better, please

Reply:
Okay, check the `routes/auth.php`. You'll see that the auth routes have already been defined there and then imported in `routes/api.php`

Reply:
Please check again, pushed a new commit